code/processes/dqe.q

```diff
@@ -1,7 +1,13 @@
 \d .dqe
 
 configcsv:@[value;`.dqe.configcsv;first .proc.getconfigfile["dqeconfig.csv"]];
+dqedbdir:@[value;`dqedbdir;`:dqedb];
```

Comment:
I think this should be driven by an environment variable rather than being a relative path.

Reply:
I made the change in setenv in torq-fsp in the branch upd_setting_dqe